sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Other Changes
 - Python 2.7 is no longer supported. Please use Python version 3.6 or later.
 - Bumped mininum dependency on `azure-core` to `>=1.24.0`.
+- Changed the trasnport from `AsyncioRequestsTransport` to `AioHttpTransport`. ([#26427](https://github.com/Azure/azure-sdk-for-python/issues/26427))
```

Comment:
I think the wording here should probably be adjusted. We did default to `AsyncioRequestsTransport` before, but we're not enforcing `AioHttpTransport` now; we're using the default aysnc transport provided by `azure-core`. That's `AioHttpTransport` right now, and we can mention that, but I feel like that's a distinction we should make. By saying we're using Core's default, we communicate that the default can change in the future if `azure-core` changes _its_ default

Reply:
That's a good point! Thanks, will update.